index.html

```diff
@@ -0,0 +1,323 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+
+
+  <section class="headline">
+    <div id="head-container">
+      <h1 class="head">
+        I'm John, <br />
+        Glad to see you!
+      </h1>
+      <p class="desc">
+        I’m a software developer! I can help you build a product, feature or
+        website Look through some of my work and experience. If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me!
+      </p>
+      <h2 class="connect">LET’S CONNECT</h2>
+
+      <ul class="social-media-icons">
+        <li>
+          <a href="#" target="_blank">
+            <img src="./images/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="images/Linkedin.png" alt="linkedIn" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="images/medium.png" alt="medium" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="images/github.png" alt="git" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="images/angelite.png" alt="angel" />
+          </a>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <main class="grid-container">
+    <section class="works-card-tonic">
+      <img class="snapshort" src="./images/snap1.png" alt="snapshoot" />
+      <div class="left-block-tonic">
+        <div class="first-container-tonic">
+          <h2 class="newme-title-tonic">Tonic</h2>
+          <div class="event2-tonic">
+            <div class="word">
+              <h3>CANOPY</h3>
+            </div>
+            <img src="./images/Counter.png" alt="counter count" />
+            <div class="backend">
+              Back End Dev
+            </div>
+            <img src="./images/Counter.png" alt="counter count" class="dot" />
+            <div class="backend">
+              2015
+            </div>
+          </div>
+        </div>
+        <p class="newme-description-tonic">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages-tonic">
+          <li class="language1">
+            html
+          </li>
+          <li class="language2">
+            css
+          </li>
+          <li class="language3">
+            javascript
+          </li>
+        </ul>
+        <div class="newmes-tonic">
+          <button type="submit" class="go-newmes">See projects</button>
+        </div>
```

Comment:
- [ ] [OPTIONAL] Unless you need to include the `div` for some structural/organizational reason you're likely better off just styling the button element. The purpose of `divs` is to allow you to group elements together. Wrapping individual elements in `divs` for the purpose of styling is largely unnecessary.

Reply:
Thank you